src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -285,6 +297,30 @@ protected void interrupted() {
         };
     }
 
+    public Command UpSpecified(double inches) {
+        return new SubsystemCommand(this) {
+            double upSpecifiedIncrement = liftEncoder.get();
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                upSpecifiedIncrement = liftEncoder.get();
+            }
+            @Override
+            protected void execute() {
+                if (upSpecifiedIncrement + 1 <=1) {
```

Comment:
You can start the liftDrive in initialize, leave execute empty. Then add an end() which calls liftDrive.set(0)

Reply:
done